tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_jumpf_execution.py

```diff
@@ -71,6 +71,44 @@ def test_jumpf_forward(
                 ),
             ],
         ),
+        Container(
+            name="equal_outputs",
+            sections=[
+                Section.Code(
+                    Op.CALLF[1] + Op.SSTORE + Op.STOP,
+                    max_stack_height=2,
+                ),
+                Section.Code(
+                    Op.JUMPF[2],
+                    code_outputs=2,
+                    max_stack_height=0,
+                ),
+                Section.Code(
+                    Op.PUSH2[value_code_worked] + Op.PUSH2[slot_code_worked] + Op.RETF,
+                    code_outputs=2,
+                    max_stack_height=2,
+                ),
+            ],
+        ),
+        Container(
+            name="compatible_outputs",
```

Comment:
I thought about "incompatible_outputs" version of this, but is already covered by https://github.com/ethereum/execution-spec-tests/blob/b22e90185bb699a5873e27e5629250807f9b3cc2/tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_jumpf_stack.py#L84 from what I can see.